### DISCUSSION TITLE
Everything is f***** up since the transfer from personal to hobby team plan !

### DISCUSSION THREAD

**Discussion by abdalem** [vercel/community#6529] [Help]
### Summary

Hi everyone,

Am I the only one who everything has been a total mess since the transfer from personnal to hobby team plan? (Don't think so given the issues). How is it even possible to transfer everything, completely crushing everyone stack, without at least a doc of what we need to do from our side ? Honestly it's a total shame.

### Steps to Reproduce

Nothing to reproduce, just trying to deploy.

**Comment by leerob** [comment]
Hey @abdalem, I'm sorry about this experience. Could you share more about what you are seeing that is off? 

https://vercel.com/changelog/2024-01-account-changes

**Reply by abdalem** [reply]
It's my domains. I am unable to deploy and link the domain. It's always marked that I have not the right to use it. Besides the fact that I transfered all my domains to the team, and for the CI, I've linked it again my project, so the ORG_ID and PROJECT_ID have been changed, and I also generated another TOKEN, but I am unable to link any alias.

**Reply by smaeda-ks** [reply]
Sorry to hear that.

re: alias assignment
I think I see the issue. Let me see if I can fix it.

As a workaround, you can try (once you moved domains to your Hobby team):
- use `vercel switch` and ensure you always run your commands in the right team context: https://vercel.com/docs/cli/switch
- use `--scope` global option: https://vercel.com/docs/cli/global-options#scope

**Reply by abdalem** [reply]
Sorry but we have decided to move out our frontends from vercel and go back to Google Cloud Run. Thanks for your time.
